index.js

```diff
@@ -66,59 +67,72 @@ function createColormap (spec) {
         alpha = [1, 1];
 
     } else {
-        alpha = clone(spec.alpha);
+        alpha = spec.alpha.slice();
     }
 
-    /*
-     * map index points from 0->1 to 0 -> n-1
-     */
+    // map index points from 0..1 to 0..n-1
     indicies = cmap.map(function(c) {
         return Math.round(c.index * nshades);
     });
 
-    /*
-     * Add alpha channel to the map
-     */
-    if (alpha[0] < 0) alpha[0] = 0;
-    if (alpha[1] < 0) alpha[0] = 0;
-    if (alpha[0] > 1) alpha[0] = 1;
-    if (alpha[1] > 1) alpha[0] = 1;
+    // Add alpha channel to the map
+    alpha[0] = clamp(alpha[0], 0, 1);
+    alpha[1] = clamp(alpha[1], 0, 1);
```

Comment:
using clamp saves 2 lines of code _at this location_ but adds a `require` line and a line to package json. Total lines saved === 0.
However the complexity of the code has increased. Not a major issue but I vote to to keep the manual clamping in place.